.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+			"type": "node",
+			"request": "attach",
+			"name": "Attach",
+			"port": 9229
+		}
+  ]
+}
```

Comment:
I don't think this should be committed. It's specific to your setup.

Reply:
Oops, that's my bad. I was debugging webpack. Sorry about that.